### ISSUE TITLE
Bug in Sample Code: Wont get_events for US Trawlers

### ISSUE THREAD

**Issue by adeloera** [GlobalFishingWatch/gfwr#65]
Running the sample code gives me an error when trying to get_events for US trawlers: 

![image](https://user-images.githubusercontent.com/32284808/182199564-8a122ec6-122d-481b-81bf-62f67af954a9.png)

**Comment by adeloera** [comment]
Despite running fine previously I've now had issues with get_events for port visits which are also present in the sample code

![image](https://user-images.githubusercontent.com/32284808/182458806-9be1c668-3a8e-4696-8f2e-43378ac04d97.png)

**Comment by natemiller** [comment]
@adeloera Thanks for the feedback. Your first error highlights some missing error handling in the `get_events` function that we will address in the next update to the `gfwr` package. Above the error message, we can see that the API request returned 0 events (`Downloading 0 events from GFW`). The error on our side is that we then attempt to tidy this empty response, throwing an error.

If you request data for more `vessel_id`s you will be more likely to identify vessels active in the selected time range and you won't see the error. The code below, selecting the first 100 vessel_ids, successfully returns 61 events.

```
usa_trawlers <- gfwr::get_vessel_info(
  query = "flag = 'USA' AND geartype = 'trawlers'",
  search_type = 'advanced',
  dataset = 'fishing_vessel'
)

# request the first 100 vessel_ids
usa_trawler_ids <- paste0(usa_trawlers$id[1:100], collapse = ',')

 get_event(event_type = 'fishing',
          vessel = usa_trawler_ids,
          start_date = '2020-01-01',
          end_date = '2020-02-01',
          key = key)
```

**Comment by adeloera** [comment]
Ah I see! That explains some other bugs I have been having where get_events fails. Thank you!

**Comment by natemiller** [comment]
@adeloera I will leave this open for now and let you know when we update the package to better handle this error.

**Comment by tclavelle** [comment]
Closed by #73

**Comment by adeloera** [comment]
Thank you!
